llvm/lib/ProfileData/Coverage/CoverageMapping.cpp

```diff
@@ -445,11 +447,11 @@ class MCDCRecordProcessor {
     //   visualize where the condition is.
     // - Record whether the condition is constant folded so that we exclude it
     //   from being measured.
-    for (const auto &B : Branches) {
-      Map[B.MCDCParams.ID] = &B;
-      PosToID[I] = B.MCDCParams.ID - 1;
-      CondLoc[I] = B.startLoc();
-      Folded[I++] = (B.Count.isZero() && B.FalseCount.isZero());
+    for (const auto *B : Branches) {
+      Map[B->MCDCParams.ID] = B;
```

Comment:
assert `B` is not null?

Reply:
This is in #78918.
I don't think `B` could be null, since `Branches` is pushed with only `Region&`.
Any other concerns?